PSCompatibilityCollector/Microsoft.PowerShell.CrossCompatibility/Utility/PowerShellExtensions.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.ObjectModel;
+using System.Management.Automation;
+using SMA = System.Management.Automation;
+
+namespace Microsoft.PowerShell.CrossCompatibility.Utility
+{
+    internal static class PowerShellExtensions
```

Comment:
I _really_ think this should be in PowerShell (it doesn't much help with down level), but it's a common pattern which we could easily improve for folks.

Reply:
Agreed. I'm not really sure how you're supposed to reuse a PowerShell instance without this. As in, I don't really understand the default scenario.